mods/gen2/scripts.js

```diff
@@ -168,7 +168,7 @@ exports.BattleScripts = {
 		}
 		accuracy = this.runEvent('Accuracy', target, pokemon, move, accuracy);
 
-		if (accuracy !== true || accuracy !== 255 && this.random(255) >= accuracy) {
+		if (accuracy !== true || accuracy == 255 && this.random(255) >= accuracy) {
```

Comment:
Why not just
`if (accuracy !== true && accuracy !== 255 && this.random(255) >= accuracy) {`
As you have it, if accuracy gets assigned a value of 255 and if this.random(255) === 255, it results in a miss. Is this intentional?

Reply:
Considers its 1/256 I'm doubtful it is

Reply:
Oh, random(n) is [0, n).

Reply:
um actually, I just realized this isn't needed. I misread how the random(n) function worked

Reply:
The main reason why I included was the way Gen II fixed the 1/256 miss glitch, but I realized random already accounted for that, which is why Gen I had to roll from 256 to replicate the glitch